templates/launch.html

```diff
@@ -1,215 +1,395 @@
+                        <label class="ons-label" for="remote-schema-survey-type">Survey Type</label>
+                        <select name="survey-types"
+                                id="remote-schema-survey-type"
+                                onchange="setSurveyType(this)"
+                                class="ons-input ons-input--select">
+                            <option selected disabled>Select Survey Type</option>
+                            {{ range $surveyType, $schemasList := .Schemas }}
+                            <option value="{{ $surveyType }}">{{ $surveyType }}</option>
+                            {{ end }}
+                        </select>
+                    </div>
+                    <div class="ons-field ons-field--inline">
+                        <label class="ons-label" for="schema-url">Schema URL</label>
+                        <input id="schema-url"
+                               name="schema_url"
+                               type="text"
+                               class="qa-schema_url ons-input ons-input--text ons-input-type__input"
+                               onchange="setSchemaUrl(this)">
+                    </div>
+                    <div class="ons-field ons-field--inline">
+                        <label class="ons-label" for="cir-schemas">CIR Schema</label>
+                        <select id="cir-schemas"
+                                name="cir_instrument_id"
+                                class="qa-cir_instrument_id ons-input ons-input--select"
+                                onchange="setCirSchema(this);">
+                            <option selected disabled>Select Schema</option>
+                            {{ range $cirSchema := .CirSchemas }}
+                            <option value="{{ $cirSchema.ID }}"
+                                    data-form-type="{{ $cirSchema.FormType }}"
+                                    data-version="{{ $cirSchema.CIVersion }}"
+                                    data-language="{{ $cirSchema.Language }}"
+                                    data-title="{{ $cirSchema.Title }}"
+                                    data-description="{{ $cirSchema.Description }}">
+                                {{ $cirSchema.FormType }} ({{ $cirSchema.Language }})
+                            </option>
+                            {{ end }}
+                        </select>
+                    </div>
+                    <div class="ons-u-mt-s">
+                        <button id="schema-url-btn"
+                                type="button"
+                                class="ons-btn qa-btn-submit-dev"
+                                onclick="loadMetadataForRemoteSchema()">
+                            <span class="ons-btn__inner"><span class="ons-btn__text">Load Metadata</span></span>
+                        </button>
+                    </div>
+                    <hr>
+                    <h3 class="ons-u-mt-m">Survey Actions</h3>
```

Comment:
This one should also be a `<h2>` because it comes under the `<h1>` in the hierarchy

Reply:
updated 👍